README.md

```diff
@@ -58,9 +58,85 @@ hzc
 
 # Print help
 hzc --help
+
+# Non-interactive mode
+hzc map --name myMap put --key myKey --value myValue
 ```
 
-### More examples
+## Connecting to Hazelcast Cloud
+
+The cluster creation and retrieving connection info can be done directly in command line using [Hazelcast Cloud CLI](https://github.com/hazelcast/hazelcast-cloud-cli).
+
+- Authenticate to Hazelcast Cloud account:
+
+  ```  
+  hzcloud login
+  -  Api Key: SAMPLE_API_KEY
+  -  Api Secret: SAMPLE_API_SECRET
+  ```
+
+- Create cluster:
+
+  ```
+  hzcloud starter-cluster create \
```

Comment:
I know this is not related to this PR but I really dislike "starter-cluster"  command name :(
Maybe I can ask If they can change it to something like trial-cluster  or dev-cluster?

Reply:
Unfortunately, this won't be easily changeable. It's related to the product name - it's not that it's a trial cluster or dev cluster, but it's using the "Hazelcast Cloud Starter" variant, rather than Hazelcast Cloud Enterprise". So it's a product naming and we can't really do anything about it without changing the product name.